paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -659,5 +659,12 @@ class Paparazzi @JvmOverloads constructor(
       } else {
         HtmlReportWriter()
       }
+
+    fun determineGifHandler(maxPercentDifference: Double = 0.1): SnapshotHandler =
+      if (isVerifying) {
+        GifSnapshotHandler(maxPercentDifference)
+      } else {
+        HtmlReportWriter()
+      }
```

Comment:
Where are you using this method?

Reply:
this will be used to override the regular snapshot handler when we actually go to use this in our test rule. Like this:
```
@get: Rule
    val paparazzi = Paparazzi(
        ...
        snapshotHandler = determineGifHandler(),
    )